src/web/modules/gateway_accounts/agent_initiated_moto.njk

```diff
@@ -18,131 +29,74 @@
     {% endif %}
   </span>
 
-  <h1 class="govuk-heading-m">Agent-initiated MOTO payments</h1>
-
-  <div>
-    <a href="/gateway_accounts/{{ account.gateway_account_id }}" class="govuk-back-link">Associated gateway account {{serviceName}} ({{ account.gateway_account_id }})</a>
-  </div>
-
-  {% if errors %}
-    {{ errorSummary({ errors: errors }) }}
-  {% endif %}
+  <h1 class="govuk-heading-m">Agent initiated MOTO</h1>
 
   <p class="govuk-body">
     Agent-initiated MOTO payments (‘telephone payment links’) allow service users with appropriate permissions to take telephone payments using a link on the admin tool dashboard.
   </p>
 
-  <p class="govuk-body">
-    For the link to appear and work, several things must be set up.
-  </p>
+  <p class="govuk-body">The gateway account must have MOTO enabled for this to work</p>
 
-  <h2 class="govuk-heading-s">The service must have agent-initiated MOTO payments enabled</h2>
-  {% if service.agent_initiated_moto_enabled %}
-    <p class="govuk-body">Agent-initiated MOTO payments are enabled for this service.</p>
-  {% else %}
-    <p class="govuk-body">Agent-initiated MOTO payments are disabled for this service.</p>
-  {% endif %}
+  <div class="govuk-inset-text">
+    {% if account.allow_moto %}
+      <p class="govuk-body">MOTO is enabled for this gateway account.</p>
+    {% else %}
+      <p class="govuk-body">MOTO is disabled for this gateway account.</p>
+    {% endif %}
+  </div>
+
+  {% if motoPaymentLinkExists %}
+    <h2 class="govuk-heading-s">Agent-initiated MOTO payment link is configured</h2>
```

Comment:
It might be useful to indicate if there is more than one as that likely indicates a mistake has been made at some point.

Reply:
Yeh makes sense, would be good to draw extra attention to this. See screenshot below for the warning text I've  now added:

<img width="653" alt="Screenshot 2024-01-12 at 11 33 39" src="https://github.com/alphagov/pay-toolbox/assets/5648592/e383c0aa-3c10-4ff8-bcf6-5a4203898df1">